nipype/interfaces/afni/preprocess.py

```diff
@@ -2406,6 +2406,170 @@ class TNorm(AFNICommand):
+                                   analysis proceeds without actual removal
+                                   of any time points -- this feature is to
+                                   keep the Spanish Inquisition happy.
+                 * The default mode is KILL !!!""",
+        argstr='-cenmode %s')
+    concat = File(
+        desc="""The catenation file, as in 3dDeconvolve, containing the
+               TR indexes of the start points for each contiguous run
+               within the input dataset (the first entry should be 0).
+               ++ Also as in 3dDeconvolve, if the input dataset is
+                  automatically catenated from a collection of datasets,
+                  then the run start indexes are determined directly,
+                  and '-concat' is not needed (and will be ignored).
+               ++ Each run must have at least 9 time points AFTER
+                  censoring, or the program will not work!
+               ++ The only use made of this input is in setting up
+                  the bandpass/stopband regressors.
+               ++ '-ort' and '-dsort' regressors run through all time
+                  points, as read in.  If you want separate projections
+                  in each run, then you must either break these ort files
+                  into appropriate components, OR you must run 3dTproject
+                  for each run separately, using the appropriate pieces
+                  from the ort files via the '{...}' selector for the
+                  1D files and the '[...]' selector for the datasets.""",
+        exists=True,
+        argstr='-concat %s')
+    noblock = traits.Bool(
+        desc="""Also as in 3dDeconvolve, if you want the program to treat
+                an auto-catenated dataset as one long run, use this option.
+                ++ However, '-noblock' will not affect catenation if you use
+                   the '-concat' option.""",
+        argstr='-noblock')
+    ort = File(
+        desc="""Remove each column in file
+                ++ Each column will have its mean removed.""",
+        exist=True,
+        argstr="-ort %s")
+    polort = traits.Int(
+        desc="""Remove polynomials up to and including degree pp.
+                ++ Default value is 2.
+                ++ It makes no sense to use a value of pp greater than
+                   2, if you are bandpassing out the lower frequencies!
+                ++ For catenated datasets, each run gets a separate set
+                   set of pp+1 Legendre polynomial regressors.
+                ++ Use of -polort -1 is not advised (if data mean != 0),
+                   even if -ort contains constant terms, as all means are
+                   removed.""",
+        argstr="-polort %d")
+    bandpass = traits.Tuple(
+        traits.Float, traits.Float,
+        desc="""Remove all frequencies EXCEPT those in the range""",
+        argstr='-bandpass %f %f')
```

Comment:
Does it make sense to use `%g`, which will generally give you nicer formatting? e.g. `2` vs `2.0000` or `1.395` vs `1.3949999999`.

There are some other instances of `%f` in here. Your call whether to switch to `%g`.